{"id": 4597, "title": "Ticket #4597: 8.0.x->8.2.2 upgrade = collector crash (NETWORK_MAX_PENDING_CONNECTS)", "description": "<blockquote>\nReport from Michael R Brady &lt;mrbrady@syr.edu&gt; of the Syracuse Orange Grid pool:\n\n<p></p><div class=\"blockquote\">\n<table width=\"95%\"><tbody><tr><td>\u00a0</td><td>\n  There was a config option the mgr side didn't like (NETWORK_MAX_PENDING_CONNECTS) causing and the Collector to crash and condor_userprio command to fail.. removing that option fixed the problem with it crashing, but now Collector reports this:</td></tr></tbody></table></div>\n\n\n<p></p><div class=\"code\">\n<pre class=\"code\">09/17/14 22:28:31 WARNING: cannot register TCP update socket from &lt;10.5.40.102:57527&gt;: file descriptor safety level exceeded:  limit 8192,  registered socket count 8186,  fd 8192\n09/17/14 22:28:31 WARNING: cannot register TCP update socket from &lt;10.5.60.9:51764&gt;: file descriptor safety level exceeded:  limit 8192,  registered socket count 8186,  fd 8192</pre></div>\n\n\n<p></p><div class=\"blockquote\">\n<table width=\"95%\"><tbody><tr><td>\u00a0</td><td>\n  Even though the file descriptior limit is set to 65000 in the OS and in the local condor_config file (MAX_FILE_DESCRIPTORS)... didn't see this in the previous release.\n</td></tr></tbody></table></div>\n</blockquote>", "remarks": "<blockquote>\nI've been fiddling with this, but I can't reproduce this now.  Can we mark this as abandoned, and revive it, if we see it again?</blockquote>", "derived_tickets": "", "attachments": "<html><head></head><body></body></html>", "check_ins": "", "type": "defect", "last_change": "2015-Oct-19 11:09", "status": "defer", "created": "2014-Sep-17 21:12", "fixed_version": "2014-Sep-17 21:12", "broken_version": "v080202", "priority": "5", "subsystem": "DaemonsCM", "assigned_to": "gthain", "derived_from": "", "creator": "pfc", "rust": "", "customer_group": "ligo", "visibility": "public", "notify": "mrbrady@syr.edu, essedore@syr.edu,pcouvare@caltech.edu", "due_date": ""}